src/main/kotlin/com/exactpro/th2/codec/fix/orchestra/FixOrchestraCodec.kt

```diff
@@ -103,8 +104,8 @@ class FixOrchestraCodec(
                 val type = checkNotNull(accessor.getMessage(name, scenario)) { "No scenario $scenario for message: $name" }
                 validator.validate(result, type)
                 dictionary.validate(result, true)
-            } catch (e: TestException) {
-                errors += e.details
+            } catch (e: TestExceptionImpl) {
+                throw ValidateException("msgType {${e.msgType}} tags {${e.tags.joinToString()}}, scenario {${e.scenario}}", e.details)
```

Comment:
I think it is better not to use curly breackets
```suggestion
                throw ValidateException("msgType '${e.msgType}' tags [${e.tags.joinToString()}], scenario '${e.scenario}'", e.details)
```